pkg/queue/health/probe.go

```diff
@@ -58,15 +57,14 @@ func TCPProbe(config TCPProbeConfigOptions) error {
 
 // HTTPProbe checks that HTTP connection can be established to the address.
 func HTTPProbe(config HTTPProbeConfigOptions) error {
+	transport := http.DefaultTransport.(*http.Transport).Clone()
+	transport.DisableKeepAlives = true
+	transport.TLSClientConfig.InsecureSkipVerify = true
```

Comment:
Should we throw this into a constant? Feels like this should be shared between executions.

Reply:
oh, that's an interesting point - yes, and also I think the fact that we're creating a new one each time (before and after this PR) is probably accidentally disabling keep alive here!

Reply:
That's a good point - yes. I got excited thinking the lack of this was probably ending up disabling keep alive, but actually looks like we explicitly disable keep alive here, even though there's code a bit lower down to close the connection so we can re-use it 🤷‍♂️. Anyway, yes :).

Reply:
I'm wondering why we disable keep-alives here :thinking:. Totally separate from this PR but maybe we should enable them?

Reply:
Yeah, Im having the _exact_ same thought. Seems entirely wrong. I have a memory of @vagababov having some reason for this, but it's _very_ possible that was in the mesh case in activator and not a thing here 🤔

Reply:
(done, btw)